### PR TITLE
Docs: zh tranlations.

### PR DESCRIPTION
**Pull request by gogoend**
Translate some untranslated page ,and fix some wrong words or expressions.

### CODE REVIEW

**Comment by Mugen87** [comment]
Hint: You might want to scan the zh-docs for the `TODO` string. When I'm updating the english docs, I always try to update the structure of the Chinese pages but of course without the actual translation. I normally use `TODO` as a placeholder.

**Comment by gogoend** [comment]
> Hint: You might want to scan the zh-docs for the `TODO` string. When I'm updating the English docs, I always try to update the structure of the Chinese pages but of course without the actual translation. I normally use `TODO` as a placeholder.

OK,got it~

**Comment by gogoend** [comment]
Besides,I found the link `https://github.com/KhronosGroup/glTF/issues/9562` at `Loading` section in manual/en/introduction/Loading-3D-models (line 100) is not available.

**Comment by Mugen87** [comment]
Oh, I think the link should be https://github.com/mrdoob/three.js/issues/9562. Can you please replace it in both versions (en, zh)?

**Comment by gogoend** [comment]
> Oh, I think the link should be #9562. Can you please replace it in both versions (en, zh)?

With pleasure.

File: docs/api/en/extras/curves/EllipseCurve.html [hunk 1]
```diff
@@ -88,12 +88,6 @@ <h3>[property:Boolean aClockwise]</h3>
 		<h3>[property:Float aRotation]</h3>
 		<p>The rotation angle of the ellipse in radians, counterclockwise from the positive X axis (optional). Default is *0*.</p>
 
-		<h3>[property:Boolean isEllipseCurve]</h3>
-		<p>
-			Used to check whether this or derived classes are ellipses. Default is *true*.<br /><br />
-
-			You should not change this, as it used internally for optimisation.
-		</p>
```
Comment (Mugen87) [inline docs/api/en/extras/curves/EllipseCurve.html]: Why have you deleted this section?

**Review by Mugen87 — CHANGES_REQUESTED** [review]

File: docs/api/en/extras/curves/EllipseCurve.html (hunk 1 above)
Comment (Mugen87) [reply docs/api/en/extras/curves/EllipseCurve.html]: Good catch :+1:

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**